dev/io.openliberty.microprofile.reactive.messaging.internal_fat/publish/servers/RxMessagingServerWithTelemetry/bootstrap.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+osgi.console=5471
+com.ibm.ws.logging.trace.specification = *=info:logservice=detail:REACTIVEMESSAGE=all:com.ibm.ws.microprofile.reactive.messaging.kafka.adapter.impl.*=FINEST:org.apache.kafka.*=all
```

Comment:
I would add the telemetry trace string in as well so that if an issue arises on that side then we don't have to add the trace later